provision/roles/docker/vars/ubuntu.yaml

```diff
@@ -0,0 +1,17 @@
+---
+docker_dependency_packages:
+  - ca-certificates
+  - curl
+  - gnupg
+  - lsb-release
+
+docker_packages:
+  - docker-ce
+  - docker-ce-cli
+  - containerd.io
+
+docker_download_base_url: "https://download.docker.com/linux"
+
+architecture_translation:
+  x86_64: amd64
+  amd64: amd64
```

Comment:
Isn't it a problem that this var is not defined for fedora?

Reply:
It's not. I'm importing the variables based on the name of the distribution `file: "{{ ansible_distribution | lower }}.yaml"`. In Fedora distributions it will be `fedora.yaml` in Ubuntu ones `ubuntu.yaml`, etc. ( https://github.com/RedHatCRE/toolbox/pull/9/files#diff-422e99f874baad3d4a72d9b903b03e23ff680f8a96300a897ddd019dbc47ab85R4 )

Reply:
Yes, but I am just not sure how Ansible processes the `provision/roles/docker/defaults/main.yml` file – it really does not complain? Then it must use some sort of lazy loading, interesting...